unit_test/test_pub_9.py

```diff
@@ -0,0 +1,19 @@
+import sys
+sys.path.insert(1, '../')
```

Comment:
Why is this necessary?

Reply:
It helps me run the tests locally but you are right I will keep it only for my local tests and remove it from the code